<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_config_obj</class>
 <widget class="QWidget" name="widget_config_obj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rele</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_rele_state"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_rele_mode"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_rele_status"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_rele_write">
        <property name="text">
         <string>Write</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_rele_read">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>state</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
